### DISCUSSION TITLE
[resolved] Numer VIN w centrum wiadomości

### DISCUSSION THREAD

**Discussion by mkulig** [allegro/allegro-api#5755] [Suggestions - resolved]
Witam,
zgodnie z prośbą z wątku https://github.com/allegro/allegro-api/issues/5754 proszę o rozważenie dodania do API (/messaging/threads/{threadId}/messages) dodatkowego pola numer VIN w pytaniach związanych z ofertami z kategorii motoryzacja, które w serwisie zostało już dodane.

![image](https://user-images.githubusercontent.com/48495083/171617664-7e9d6ec5-2dee-44b5-9110-2271994d2a38.png)

**Comment by MaciejFrackowiak** [comment]
Numer VIN dostępny jest już w Centrum wiadomości - https://github.com/allegro/allegro-api/issues/5964.
